client/post-editor/post-editor.jsx

```diff
@@ -379,6 +379,13 @@ const PostEditor = React.createClass( {
 			this.setState( this.getInitialState(), function() {
 				this.refs.editor.setEditorContent( '' );
 			} );
+		} else if ( this.state.isNew && this.state.hasContent && ! this.state.isDirty ) {
+			// Is a copy of an existing post.
+			// When copying a post, the created draft is new and the editor is not yet dirty, but it already has content.
```

Comment:
Thanks for the detailed comment here. Based on the new Flux action creator you'd implemented above, I'd have expected the call to `edit` to cause the post to become dirty. Is that not the case?